From 2989d010b4b0d33ec8e0f4fafbd9c5cff34f2283 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Tue, 7 Jan 2020 18:47:45 +0100
Subject: [PATCH] PONRTSYS-5630: Update IFLA_BRPORT to recent kernel

This adds the additional attributes supported in the IFLA_BRPORT in
kernel 5.4 to our kernel 4.9.

Newly added attributes will not change order of the existing options
and will also get the same number in the array, this way the user space
iproute2 application should be compatible to the LGM kernel and the
PRX300 kernel.
---
 include/uapi/linux/if_link.h | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/include/uapi/linux/if_link.h b/include/uapi/linux/if_link.h
index b4fba662cd32..89389d8403ad 100644
--- a/include/uapi/linux/if_link.h
+++ b/include/uapi/linux/if_link.h
@@ -319,6 +319,13 @@ enum {
 	IFLA_BRPORT_MULTICAST_ROUTER,
 	IFLA_BRPORT_PAD,
 	IFLA_BRPORT_MCAST_FLOOD,
+	IFLA_BRPORT_MCAST_TO_UCAST,
+	IFLA_BRPORT_VLAN_TUNNEL,
+	IFLA_BRPORT_BCAST_FLOOD,
+	IFLA_BRPORT_GROUP_FWD_MASK,
+	IFLA_BRPORT_NEIGH_SUPPRESS,
+	IFLA_BRPORT_ISOLATED,
+	IFLA_BRPORT_BACKUP_PORT,
 	__IFLA_BRPORT_MAX
 };
 #define IFLA_BRPORT_MAX (__IFLA_BRPORT_MAX - 1)
